src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclint/resources/doclint.properties

```diff
@@ -27,12 +27,12 @@ dc.anchor.already.defined = anchor already defined: "{0}"
 dc.anchor.value.missing = no value given for anchor
 dc.attr.lacks.value = attribute lacks value
 dc.attr.not.number = attribute value is not a number
-dc.attr.not.supported.html4 = attribute not supported in HTML4: {0}
 dc.attr.not.supported.html5 = attribute not supported in HTML5: {0}
 dc.attr.obsolete = attribute obsolete: {0}
 dc.attr.obsolete.use.css = attribute obsolete, use CSS instead: {0}
 dc.attr.repeated = repeated attribute: {0}
-dc.attr.table.border.html5 = attribute border for table only accepts "" or "1", use CSS instead: {0}
+dc.attr.img.border = attribute border for img only accepts "0", use CSS instead: {0}
+dc.attr.table.border = attribute border for table only accepts "" or "1", use CSS instead: {0}
```

Comment:
I suggest quoting the attribute name "border"

Reply:
Ok.